### ISSUE TITLE
Add 'Next time' to Disk not eject properly toast

### ISSUE THREAD

**Issue by mpvader** [victronenergy/venus#132]
change text to:

```
This can cause data corruption. Next time, press Eject in
the VRM settings menu before removing the disk.
```

![image](https://cloud.githubusercontent.com/assets/5200296/24095916/8d2d868c-0d5f-11e7-906d-4098af47b2dd.png)

**Comment by jepefe** [comment]
Done in improve_sdcard_notifications branch.

**Comment by jhofstee** [comment]
included in v2.07~1
